hydra-node/test/Hydra/HeadLogicSpec.hs

```diff
@@ -343,14 +347,15 @@ isAckSn = \case
 
 inInitialState :: [Party] -> HeadState SimpleTx
 inInitialState parties =
-  InitialState
-    { parameters
-    , pendingCommits = Set.fromList parties
-    , committed = mempty
-    , previousRecoverableState = idleState
-    , chainState = SimpleChainState{slot = ChainSlot 0}
-    , headId = testHeadId
-    }
+  Initial
+    InitialState
```

Comment:
Remark: It makes me sad after all those years Haskell still does not have first-class support for proper (Structural) record types